index.html

```diff
@@ -40,5 +40,138 @@ <h1>Hey there. I'm Amaury <span>I'm a software developer</span></h1>
+              <li>Rubi on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#" class="green-button">See Project</a>
+          </div>
+        </div>
+        <div class="work-card">
+          <img src="images/projects-screens/app_screenshot.png" alt="project screenshot"/>
+          <div class="work-card-info">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Rubi on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#" class="green-button">See Project</a>
+          </div>
+        </div>
+        <div class="work-card">
+          <img src="images/projects-screens/app_screenshot.png" alt="project screenshot"/>
+          <div class="work-card-info">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Rubi on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#" class="green-button">See Project</a>
+          </div>
+        </div>
+        <div class="work-card">
+          <img src="images/projects-screens/app_screenshot.png" alt="project screenshot"/>
+          <div class="work-card-info">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Rubi on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#" class="green-button">See Project</a>
+          </div>
+        </div>
+        <div class="work-card">
+          <img src="images/projects-screens/app_screenshot.png" alt="project screenshot"/>
+          <div class="work-card-info">
+            <h3>Multi-Post Stories Gain+Glory</h3>
+            <ul>
+              <li>Rubi on rails</li>
+              <li>css</li>
+              <li>JavaScript</li>
+              <li>html</li>
+            </ul>
+            <a href="#" class="green-button">See Project</a>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me-section">
+      <div class="about-me-header">
+        <h2 class="section-headline">About me</h2>
+        <p class="paragraph-text">
+          I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.
+        </p>
+        <a class="green-button" href="#">Get My Resume</a>
+      </div>
+      <div class="about-me-cards-container">
+        <div class="knowledge-card">
+          <div class="knowledge-icon-container">
+            <img src="images/icons/languages-icon.svg" alt="Languajes"/>
+          </div>
+          <h3>Languajes</h3>
+          <ul class="knowledge-card-items">
+            <li>JavaScript</li>
```

Comment:
- [ ] Please fix the spelling here 
![image](https://user-images.githubusercontent.com/74744787/159029689-31e8822e-f29a-4e6c-abe9-f7c10d85e85d.png)